cli/cmd/install.go

```diff
@@ -229,6 +230,9 @@ func checkGatewayAPICRDs(ctx context.Context, k8sAPI *k8s.KubernetesAPI) (Gatewa
 				return Linkerd, nil
 			}
 			result = External
+			if !crdIncludesV1(crd) {
+				return result, fmt.Errorf("the %s CRD is missing the v1 version, please upgrade to Gateway API v1.0.0 or later", name)
```

Comment:
According to the table we've been working with, GRPCRoute in Gateway API v1 standard is missing and it's at v1alpha2 in experimental. Probably better to just point to the same version we use in the error above (v1.2.1), and moving it into a const.

Reply:
Good catch!  I've updated it to v1.1.1 since this is effectively our minimum support version.  I think it'd be a bit confusing to suggest that we require v1.2.1 when Linkerd itself will only install v1.1.1.